src/schema/entities.yaml

```
@@ -0,0 +1,127 @@
+sub:
+  name: Subject
+  description: "A person or animal participating in the study."
+  format: label
+ses:
+  name: Session
+  description: "A logical grouping of neuroimaging and behavioral data
+                consistent across subjects.
+                Session can (but doesn't have to) be synonymous to a visit in a
+                longitudinal study.
+                In general, subjects will stay in the scanner during one session.
+                However, for example, if a subject has to leave the scanner room
+                and then be re-positioned on the scanner bed, the set of MRI
+                acquisitions will still be considered as a session and match
+                sessions acquired in other subjects.
+                Similarly, in situations where different data types are obtained
+                over several visits (for example fMRI on one day followed by DWI
+                the day after) those can be grouped in one session.
+                Defining multiple sessions is appropriate when several identical
+                or similar data acquisitions are planned and performed on all
+                -or most- subjects, often in the case of some intervention between
+                sessions (e.g., training)."
```

I believe YAML supports multi-line strings a bit more naturally: For your consideration:

```suggestion
  description: |
    A logical grouping of neuroimaging and behavioral data consistent across subjects.
    Session can (but doesn't have to) be synonymous to a visit in a longitudinal study.
    In general, subjects will stay in the scanner during one session.
    However, for example, if a subject has to leave the scanner room and then be
    re-positioned on the scanner bed, the set of MRI acquisitions will still be considered
    as a session and match sessions acquired in other subjects.
    Similarly, in situations where different data types are obtained over several visits
    (for example fMRI on one day followed by DWI the day after) those can be grouped in one
    session.
    Defining multiple sessions is appropriate when several identical or similar data
    acquisitions are planned and performed on all -or most- subjects, often in the case of
    some intervention between sessions (e.g., training)."
```